pkg/card/cards.go

```diff
@@ -53,38 +53,38 @@ var commonCards = []models.Card{
 		Cost:        0,
 		Rarity:      models.Common,
 		Actions: []models.IAction{
-			action.Draw {Amount: 1},
+			action.Draw{Amount: 1},
 		},
 	},
 	{
-		ID: 		6,
-		Name: 		"Smart fist",
-		Description:"This fist is smart",
-		Cost:		4,
-		Rarity: 	models.Common,
+		ID:          6,
```

Comment:
same question for indentation ? I'm not used to go so is there a prettier config file that follows good practice ?